_includes/layout/base/html-head.html

```diff
@@ -16,6 +16,8 @@
 {% if page.lang == 'bg' or page.lang == 'el' or page.lang == 'ko' or page.lang == 'hi' or page.lang == 'pl' or page.lang == 'sl' or page.lang == 'ro' or page.lang == 'ru' or page.lang == 'tr' or page.lang == 'uk' or page.lang == 'zh_CN' or page.lang == 'zh_TW' %}<link rel="stylesheet" href="/css/sans.css">{% endif %}
 <script type="text/javascript" src="/js/base.js"></script>
 {% if page.id != 'download' %}<script type="text/javascript" src="/js/main.js"></script>{% endif %}
+<script src="/js/jquery/jquery-1.11.2.min.js"></script>
```

Comment:
> Also, keep in mind this is version 1.11.2 which dates back to December 2014. "Old" is an understatement.

The "old" jQuery is sitting in the repository for two years. It makes absolute sense to upgrade it, however it's out of scope of this PR.

> I'd still try to deliver the smallest payload possible.

That's great! Good for you and I'm all in. Furthermore I totally agree with each point that you raised (except `requestAnimationFrame` and `setInterval` for the animations, they are just simple CSS transitions).

TL;DR:
* No doubts there are a lot of things that can be optimized and you always can open a separate PR aside/later to address those issues and do what.
* The work done here is basically just working with what is already present in the repository. 

Agree, QR generator could be used without jQuery and DOM modifications could be done with native document and node methods.

If anybody else has the same concerns about performance and minimal footprint of the website, it won't be hard to get rid of jQuery.